content/blog/simplify-embedded-projects-with-esp-bsp/index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Simplify Your Embedded Projects with ESP-BSP"
+date: 2024-06-18
+showAuthor: false
+authors:
+    - "juraj-michalek"
+tags: ["Embedded Systems", "ESP32", "ESP32-S3", "Espressif", "BSP"]
+---
+
+# Simplify Your Embedded Projects with ESP-BSP
+
+## Introduction
+
+Are you a maker or an embedded systems enthusiast looking to create applications that work seamlessly across different ESP32 development boards? Whether you’re using the ESP-Wrover-Kit, M5Stack-CoreS3, ESP32-S3-BOX-3, or other compatible boards, the ESP Board Support Package (ESP-BSP) makes your life easier. In this article, we’ll walk you through how to get started with ESP-BSP, enabling you to focus on your project’s functionality without worrying about hardware differences.
+
+## What is ESP-BSP?
+
+ESP-BSP is a collection of Board Support Packages specifically designed for Espressif’s development boards. It provides a standardized interface, allowing you to develop applications that are easy to port between different boards. By using ESP-BSP, you can:
```

Comment:
I am not sure I would emphasize specifically this part. It is not a goal of ESP-BSP to offer a standardized interface between the boards, or to make porting applications between boards easier. The main goal of ESP-BSP is to offer a convenient way to start a project for a particular development board, without having to look for drivers and other details for that specific board.

Due to the fact that now we have common examples between BSPs, the APIs of different BSPs also have a lot in common. But this is more of a side effect rather than the project goal.

I would suggest rewriting this paragraph focusing on the ease of getting started, and not promising standardized APIs or portability between boards.

Reply:
Thanks. Text updated.